app/views/chargeback_rate/_tree.html.haml

```diff
@@ -0,0 +1 @@
+= render(:partial => "shared/tree", :locals => {:tree => @tree, :name => @tree.name})
```

Comment:
This won't make @brumik happy :laughing: not really this one, but that his PR is still not in :confused: